{"name": "test_div[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert -0.5 == 0.3", "trace": "self = <test_calc.TestCalc object at 0x0000024BCD5D0460>, get_calc = <calc.calculator object at 0x0000024BCD5CE5B0>, get_datas1 = [0.1, -0.2, 0.3]\n\n    @allure.story(\"测试除法\")\n    @pytest.mark.div\n    @pytest.mark.run(order=2)\n    def test_div(self, get_calc, get_datas1):\n        #调用通过div方法\n        with allure.step(\"计算两个数相除\"):\n            result = get_calc.div(get_datas1[0], get_datas1[1])\n        # 得到结果之后，需要写断言\n>       assert result == get_datas1[2]\nE       assert -0.5 == 0.3\n\ntest_calc.py:30: AssertionError"}, "steps": [{"name": "计算两个数相除", "status": "passed", "start": 1613654698693, "stop": 1613654698693}], "attachments": [{"name": "stdout", "source": "5a2caff3-dbd1-495c-9134-b8ffd64a2e9d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_datas1", "value": "[0.1, -0.2, 0.3]"}], "start": 1613654698693, "stop": 1613654698693, "uuid": "3e53c69b-3c92-4bc2-9a07-1aef50ca5661", "historyId": "a489c2561226748904bc8930ccd3bcfd", "testCaseId": "4a57f615ae930c12d705a9fc7a66b88d", "fullName": "test_calc.TestCalc#test_div", "labels": [{"name": "feature", "value": "测试计算器"}, {"name": "story", "value": "测试除法"}, {"name": "tag", "value": "div"}, {"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-5AM7K0K"}, {"name": "thread", "value": "14824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}